INSTALL.md

```diff
@@ -119,16 +128,17 @@ To build the JavaScript files for production, run the command:
 
     yarn build
 
-## Testing Your Setup
-Once you have completed the steps above you will want to test your setup. Unfortunately, UDOIT is an LTI tool that can only fully run within the LMS. You will need to complete the steps in the [INSTALL_CANVAS.md](INSTALL_CANVAS.md) or [INSTALL_D2L.md](INSTALL_D2L.md) to test UDOIT fully.
-
-However, UDOIT does have one URL that is publicly available outside of the LMS. To test your server setup point your browser to:
+## Testing successful installation
+While UDOIT only functions fully within an LMS, there is one URL publicly available you can access to test your server setup.
 
     <BASE_URL>/lti/config
 
 For example, if you are setting this up on your local computer via Docker, it may look like:
 
     http://127.0.0.1:8000/udoit3/lti/config
 
-## Configuring Your LMS
-You will need to complete the steps in the [INSTALL_CANVAS.md](INSTALL_CANVAS.md) or [INSTALL_D2L.md](INSTALL_D2L.md) to configure UDOIT to work within your LMS.
+
+## Connecting UDOIT to an LMS
+UDOIT is an LTI tool that functions fully within an LMS. 
```

Comment:
```suggestion
UDOIT is an LTI tool that functions fully within an LMS. To configure it fully within your LMS, follow the installations instructions below that apply to you.
```
Way more verbose, so I'm open to pushback on this one.

Reply:
I don't think either matters - I'd argue that the following step implies choosing either one, but doesn't hurt to add your suggestion. Adding it in a commit.